modGui/src/org/aion/wallet/util/AddressUtils.java

```diff
@@ -0,0 +1,21 @@
+package org.aion.wallet.util;
+
+import org.aion.base.util.TypeConverter;
+
+public class AddressUtils {
+
+    public static boolean isValid(final String address) {
+        return address != null && !address.isEmpty() && isAionAddress(address);
+    }
+
+    public static boolean equals(final String addrOne, final String addrTwo) {
+        return TypeConverter.toJsonHex(addrOne).equals(TypeConverter.toJsonHex(addrTwo));
+    }
+
+    private static boolean isAionAddress(final String address) {
+        final boolean isFull = address.startsWith("0xa") && address.length() == 66;
+        final boolean isStripped = address.startsWith("a") && address.length() == 64;
```

Comment:
in the current definition, the address should start with "0xa0" or "a0", also make sure the input string is lowercase

Reply:
Added lowercase restriction and a few test cases for this method